sycl/source/detail/scheduler/graph_builder.cpp

```diff
@@ -698,8 +734,12 @@ Scheduler::GraphBuilder::addCG(std::unique_ptr<detail::CG> CommandGroup,
     NewCmd->addDep(e);
   }
 
+  if (CGType == CG::CGTYPE::CODEPLAY_HOST_TASK)
+    addEmptyCmd(NewCmd.get(), NewCmd->getCG()->MRequirements, Queue);
```

Comment:
This function allocates and returns a new EmptyCommand object. It looks like this call just discards the returned pointer to the new object. Is this a leak?